hightime/_datetime.py

```diff
@@ -118,41 +120,51 @@ def __new__(cls, *args, **kwargs):
 
     @property
     def femtosecond(self):
-        """femtosecond (0-999999999)"""
+        """The number of femtoseconds, between 0 and 999999999, inclusive."""
         return self._femtosecond
 
     @property
     def yoctosecond(self):
-        """yoctosecond (0-999999999)"""
+        """The number of yoctoseconds, between 0 and 999999999, inclusive."""
         return self._yoctosecond
```

Comment:
I changed these due to pydocstyle `D402	First line should not be the function’s “signature”` but after seeing the Sphinx output I'm going to revert these and add a noqa, since all of the `datetime` attributes are documented like this.